src/features/PesterTests.ts

```diff
@@ -15,6 +15,17 @@ export class PesterTestsFeature implements IFeature {
     private languageClient: LanguageClient;
 
     constructor(private sessionManager: SessionManager) {
+        this.command = vscode.commands.registerCommand(
+            "PowerShell.RunPesterTestsFromFile",
+            (uriString, runInDebugger, describeBlockName?) => {
+                this.launchTests(vscode.window.activeTextEditor.document.uri, false, describeBlockName);
```

Comment:
Maybe worth documenting the `true` and `false` somehow? I assume it's "start debugger" or similar?

Reply:
It's the `runInDebugger` parameter, which is defined on `launchTests`, isn't that sufficient? I simplified and minimised the lambda expression, maybe it is clearer now?